document/CoreFormat.mdk

```diff
@@ -413,25 +414,58 @@ along with its version and any other useful identifying information, such as its
 `"CodeScanner 2.0, Developer Preview (en-US)"`
 ~
 
-### `version` property { #tool-version }
+### `semanticVersion` property { #tool-semanticVersion }
 
-A `tool` object should contain a property named `version` whose value is a string containing the tool version.
-If present, the string should be in the format specified by Semantic Versioning 2.0.0.
+In a log file produced by a static analysis tool, a `tool` object shall contain a property named `semanticVersion`
+whose value is a string containing the tool version in the format specified by Semantic Versioning 2.0.0 ("SemVer").
 
-~ IsoNote
+~ IsoNumberedExample { @IsoNumberedExample=1 }
+`"2.0.0-beta.1"`
+~
+
+~ IsoNumberedNote { @IsoNumberedNote=1 }
 Semantic versions have the property of being sortable in chronological order of release.
+The presence of the `semanticVersion` property allows results management systems to (for example)
+restrict the results they display to versions newer than a specified version,
+or to restrict the results to a particular major version.
 ~
 
-~ IsoExample
-`"1.0.0-beta"`
+If the tool does not natively present its version string in SemVer format,
+it shall synthesize a SemVer string to populate the `semanticVersion` property. 
```

Comment:
should you say something along the lines of 'otherwise, it should recap its native version string in semanticVersion'. although you said 'shall' earlier, it's worth restating here, do you think? also, should we mention the converter case? i.e., don't synthesize